solr/solrj/src/java/org/apache/solr/common/util/Utils.java

```diff
@@ -303,6 +303,11 @@ public static Object fromJSON(byte[] utf8) {
   }
 
   public static Object fromJSON(byte[] utf8, int offset, int length) {
+    return fromJSON(utf8, offset, length, STANDARDOBJBUILDER);
+  }
+
+  public static Object fromJSON(
```

Comment:
This looks like it is only called with the customized object builder from `ClusterState.createFromJSON` which is only called from BackupManager. That feels like we're missing some usages.